--- cctools.orig/include/mach/ppc/_structs.h	2010-05-30 11:47:48.055772569 +0000
+++ cctools/include/mach/ppc/_structs.h	2010-05-31 12:11:06.905976295 +0000
@@ -136,7 +136,6 @@
 };
 #endif /* __DARWIN_UNIX03 */
 
-#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
 
 #pragma pack(4)			/* Make sure the structure stays as we defined it */
 
@@ -236,7 +235,6 @@
 
 #pragma pack()
 
-#endif /* (_POSIX_C_SOURCE && !_DARWIN_C_SOURCE) */
 
 /* This structure should be double-word aligned for performance */
 
@@ -353,7 +351,6 @@
 };
 #endif /* __DARWIN_UNIX03 */
 
-#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
 #if __DARWIN_UNIX03
 #define _STRUCT_PPC_EXCEPTION_STATE64	struct __darwin_ppc_exception_state64
 _STRUCT_PPC_EXCEPTION_STATE64
@@ -385,7 +382,6 @@
 #endif
 };
 #endif /* __DARWIN_UNIX03 */
-#endif /* (_POSIX_C_SOURCE && !_DARWIN_C_SOURCE) */
 
 #pragma pack()
 
