--- cctools.orig/include/mach/i386/_structs.h	2010-05-30 11:47:48.058904533 +0000
+++ cctools/include/mach/i386/_structs.h	2010-05-31 12:08:57.448711847 +0000
@@ -96,18 +96,14 @@
 				__precis	:1,
 						:2,
 				__pc		:2,
-#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
 #define FP_PREC_24B		0
 #define	FP_PREC_53B		2
 #define FP_PREC_64B		3
-#endif /* !_POSIX_C_SOURCE || _DARWIN_C_SOURCE */
 				__rc		:2,
-#if !defined(_POSIX_C_SOURCE) || defined(_DARWIN_C_SOURCE)
 #define FP_RND_NEAR		0
 #define FP_RND_DOWN		1
 #define FP_RND_UP		2
 #define FP_CHOP			3
-#endif /* !_POSIX_C_SOURCE || _DARWIN_C_SOURCE */
 					/*inf*/	:1,
 						:3;
 };
